js/docx/smoke/api_stroke/to_json.js

```diff
@@ -0,0 +1,29 @@
+builder.CreateFile("docx");
+    let oDocument = Api.GetDocument();
+    let oParagraph = oDocument.GetElement(0);
+    let oRGBColor = Api.CreateRGBColor(255, 111, 61);
+    let oFill = Api.CreateSolidFill(oRGBColor);
+    let oStroke = Api.CreateStroke(5 * 36000, Api.CreateSolidFill(Api.CreateRGBColor(51, 51, 51)));
+    let jsonStroke = oStroke.ToJSON();
+        oParagraph.AddText(jsonStroke);
+        GlobalVariable["JSON_Stroke"] = jsonStroke;
+builder.CloseFile();
+
+//////////////////////
+
+builder.CreateFile("docx");
+    let oDocument = Api.GetDocument();
+    let oParagraph1 = oDocument.GetElement(0);
+    let jsonStroke = GlobalVariable["JSON_Stroke"];
+    let oStrokeFromJSON = Api.FromJSON(jsonStroke);
+        oParagraph1.AddText(jsonStroke);
+        oDocument.Push(oParagraph1);
+    let oParagraph2 = Api.CreateParagraph();
+    let oRGBColor2 = Api.CreateRGBColor(255, 213, 191);
+    let oFill2 = Api.CreateSolidFill(oRGBColor2);
```

Comment:
In this js context, you have a single Fill. I think there is no need to apply an index to the name